teslajsonpy/controller.py

```diff
@@ -1012,7 +1011,7 @@ def get_climate_params(self, car_id: Text = None, vin: Text = None) -> Dict:
         return self.__climate
 
     def set_climate_params(
-        self, car_id: Text = None, vin: Text = None, params: Dict = {}
+        self, car_id: Text = None, vin: Text = None, params: Dict = None
```

Comment:
This is a good fix across the board given the default {} can result in [strange behavior](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python) since it's passing a reference to a dict. Typically you also have to do:
```python
params = params or {}
```
So it'll assign a new dict as the default. However, it might not be necessary if self.__climate[vin] accessors handle the None case.

Reply:
I'll look into this.  I don't think there is a problem, but I'll check if we try to iterate the dicts somewhere and maybe just add the extra line as suggested as a security measure.